Modules/Core/Controllers/GetController.php

```
@@ -2,91 +2,113 @@
         header('Content-Type: application/json; charset=utf-8');
-        if ($url_key && ! $result = (new UploadsService())->getFiles($url_key)) {
-            exit('{}');
+        
+        if ($url_key && !$result = $this->uploadService->getFiles($url_key)) {
+            echo '{}';
+            exit;
         }
+        
         echo json_encode($result);
         exit;
     }
```

PHPMD (2.15.0)

Avoid assigning values to variables in if clauses and the like (line '52', column '26'). (undefined)

(IfStatementAssignment)